docs/adr/adr-006-fraud-service.md

```diff
@@ -117,6 +118,7 @@ type ProofType int
 // Proof is a generic interface that will be used for all types of fraud proofs in the network.
 type Proof interface {
    Type() ProofType
+   Hash() string
```

Comment:
In the interface it might be a bit unclear what this hash refers to. Is it the Hash() (summary) of the Proof? That is what you'd usually expect from a Hash() method.